mock/operations/health.ts

```diff
@@ -0,0 +1,13 @@
+import * as express from 'express';
+import { healthList } from '../data';
+
+export function healthGet(req: express.Request, res: express.Response): express.Response {
+  const { id } = req.params;
+  const law = healthList.content?.find((el) => el.id === +id);
```

Comment:
`+id` -> `parseInt()`?

Reply:
To też już kod z developa. W sumie co kto lubi https://stackoverflow.com/questions/17106681/parseint-vs-unary-plus-when-to-use-which/17106702

Reply:
IMO kod ma być przede wszystkim czytelny, ten wątek ma już kilka lat na SO, mogło się dużo zmienić :)

Reply:
"As far as I know the first two are completely equivalent, and the choice between them is a matter of taste. (Personally I prefer the unary + because it's more concise, and well understood by most JS developers.)"

Reply:
mogę to globalnie zmienić nie ma sprawy

Reply:
to kwestia jakiś globalnych ustaleń konwencji, trzeba się zgodzić na coś i tego trzymać :) jak wszystkim pasuje `+id` to może być, jak wolelibyśmy `parseInt()` to zmieńmy

Reply:
Nie mam tu jakiegoś mocnego zdania, ale jeśli chcemy unikać `+id`, to trzeba je zastąpić `Number(x)`, nie `parseInt(x)`, bo `parseInt` to zupełnie inna funkcja: po pierwsze, `parseInt` zwraca tylko liczby całkowite, a po drugie, sparsuje dowolny dziwny tekst pod warunkiem, że zaczyna się od liczby, odrzucając suffix.

Reply:
Number już używałem w projekcie, jestem za.

Reply:
podmieniłem